src/app/frontend/common/components/ingressrulelist/component.ts

```diff
@@ -49,12 +51,24 @@ export class IngressRuleFlatListComponent {
             } as IngressRuleFlat)
         )
       )
-    );
+    ) as IngressRuleFlat[];
+
+    ingressRuleList.forEach((ingressRule, _) => {
+      tlsList.forEach((tls, _) => {
+        tls.hosts.forEach((tlsHost, _) => {
+          if (tlsHost === ingressRule.host) {
+            ingressRule.tlsSecretName = tls.secretName;
```

Comment:
3 loops nested in each other don't look good. Can we improve it by using set or a map?

Reply:
I removed one loop and now I used `every`, which means I can break out of my loops whenever I get the answer I needed. How about that ?